util/validators.ts

```diff
@@ -15,3 +29,37 @@ export function validateApiList(val: string): string {
 
   return val;
 }
+
+export function validatePhoneFormat(number: string): string {
+  let result: boolean;
+  try {
+    result = VALID_PHONE_REGEX.test(number);
+  } catch {
+    throw new Error('it was unable to process the provided data');
+  }
+
+  if (!result) {
+    throw new Error('Phone number format invalid.');
```

Comment:
would it make sense to give back a message of a given format(s) that are valid?  I know many times I expect to type in a phone number like this :  3034558666 and I expect a web page to be able to format that. It looks like that would error as invalid right now?

Reply:
That would be a valid format. You can see some examples of the regex being used here https://regexr.com/6281m . It might be better to change the message to `Phone number format invalid. Please ensure number is in NANP format.`

Reply:
or perhaps just give a block of examples. IE:
```
Valid formats:
222-333-4444
(222) 333-4444
2223334444
```

Reply:
error message has been updated in latest commit.
 ```
{
    "errors": [
        "\"phoneNumber\" failed custom validation because phone number format invalid. Valid format examples: 222-333-4444, (222) 333-4444, 2223334444"
    ]
}
```

Reply:
thanks!